controllers/provider-azure/charts/provider-azure/templates/rbac.yaml

```diff
@@ -9,11 +9,15 @@ rules:
 - apiGroups:
   - extensions.gardener.cloud
   resources:
+  - backupbucket
+  - backupbucket/status
+  - backupentry
+  - backupentry/status
```

Comment:
Use

```yaml
  - backupbuckets
  - backupbuckets/status
  - backupentries
  - backupentries/status
```

like in https://github.com/gardener/gardener-extensions/commit/1cbdc1d07e75fb6f5a978b33c27c2127016e6ce6